common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -744,7 +747,12 @@ def create_response_element(self, **kwargs):
                 hintgroup_element.set("hintfn", hint_fn)
 
         for additional_answer in additional_answers:
-            etree.SubElement(response_element, "additional_answer").text = additional_answer
+            additional_node = etree.SubElement(response_element, "additional_answer")  # pylint: disable=no-member
```

Comment:
I actually think there was - and perhaps still is - a bug in Pylint here regarding members of modules resulting in "etree.<valid_function_name>" raising a no-member error.

We've recently upgraded to Pylint 1.4.1, which claims to have resolved the issue according to a maillist post. Once you've rebased atop current master, can you try removing it (and the one at L754) and seeing if the pylint error is still raised? If so, no worries, keep the pragma in.